server/lib/topic_cache.py

```diff
@@ -121,8 +122,9 @@ def load_files(fpath_list: List[str], name_overrides: Dict) -> TopicCache:
     dcid = node['dcid'][0]
     typ = node['typeOf'][0]
     name = node.get('name', [''])[0]
-    if name_overrides.get(dcid):
-      name = name_overrides.get(dcid)
+    name_override_info = name_overrides.get(dcid)
+    if name_override_info and not dc in name_override_info['blocklist']:
```

Comment:
nit:  be defensive name_override_info.get('blocklist', []), and below

Reply:
done